CODEOWNERS

```diff
@@ -1,4 +1,5 @@
 * @defenseunicorns/zarf @dgershman
+* @defenseunicorns/zarf @dhilgaertner
```

Comment:
We can move all the names to line 1 instead of repeating defenseunicorns/zarf

Reply:
Done